ww3tools/wread.py

```diff
@@ -1,3 +1,4 @@
+<<<<<<< HEAD
```

Comment:
@Ghazal-Mohammadpour  Thanks for merging develop into your branch.  It looks as if some of the merge statements are still here.  Have you had a chance to review the code and test it again since merging in develop?